Kernel/Heap/Heap.h

```diff
@@ -20,22 +20,18 @@ class Heap {
 
     struct AllocationHeader {
         size_t allocation_size_in_chunks;
-#if ARCH(X86_64)
-        // FIXME: Get rid of this somehow
-        size_t alignment_dummy;
-#endif
-        u8 data[0];
+        alignas(16) u8 data;
```

Comment:
Please do `alignas(void*)` in these cases, that makes it easier to understand, why it is needed that way, and saves those few Bytes in 32-bit mode